src/argilla/utils/dependency.py

```diff
@@ -63,22 +63,29 @@ def _compare_versions(
         )
 
 
-def require_version(requirement: str, func_name: Optional[str] = None) -> None:
+def require_version(requirement: Union[str, List[str]], func_name: Optional[str] = None) -> None:
     """
     Perform a runtime check of the dependency versions, using the exact same syntax used by pip.
     The installed module version comes from the *site-packages* dir via *importlib_metadata*.
 
     Args:
-        requirement (`str`): pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
+        requirement (`str` or `List[str]`): pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
```

Comment:
I think we can rename this arg to `requirements` and remove the type-hints from the docstring, as those are automatically generated based on the function type-hints 👍🏻 

```suggestion
        requirements: pip style definition of dependencies, e.g. "tokenizers==0.9.4", "tqdm>=4.27", "numpy". Can either be a single dependency or a list of dependencies.
```

Reply:
Done! I missed the docstrings update. Regarding the autogenerated type-hints, I didn't know that, thank you!